cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -66,16 +70,37 @@ void Model::setInstancedAttributes(const std::shared_ptr<garrow::Table>& attribu
 }
 
 void Model::setUniforms(const std::vector<std::shared_ptr<garrow::Array>>& uniforms) {
-  std::vector<BindingInitializationHelper> bindings;
+  std::vector<std::shared_ptr<BindingInitializationHelper>> bindings;
   for (uint32_t i = 0; i < uniforms.size(); i++) {
-    auto binding = BindingInitializationHelper{i, uniforms[i]->buffer(), 0, this->_uniforms[i].elementSize};
-    bindings.push_back(binding);
+    bindings.push_back(std::make_shared<BindingInitializationHelper>(i, uniforms[i]->buffer(), 0,
```

Comment:
Double check that we understand how to bind multiple uniform buffers? Could test by splitting animometer uniforms into two buffers?